pocs/tests/test_pocs.py

```diff
@@ -177,11 +187,13 @@ def test_is_weather_safe_no_simulator(pocs, db):
     assert pocs.is_weather_safe() is False
 
 
-def test_run_wait_until_safe(db):
+def test_run_wait_until_safe(db, observatory):
     os.environ['POCSTIME'] = '2016-08-13 23:00:00'
 
     def start_pocs():
-        pocs = POCS(simulator=['camera', 'mount', 'night'],
+        observatory.config['simulator'] = ['camera', 'mount', 'night']
+
+        pocs = POCS(observatory,
                     messaging=True, safe_delay=15)
```

Comment:
FWIW, looks like safe_delay is one of several undocumented parameters to POCS.

Reply:
Created issue #198